frontend/src/components/widgets/BaseButtonLink.js

```diff
@@ -1,21 +1,24 @@
 import React from "react";
 import { Link } from "react-router-dom";
-
-import {
-    Button,
-  } from "@material-ui/core";
-
-export default ({to, label,selected}) => {
-
-    const variant = selected ? "contained" : "outlined"
-    return (
-
-        <Link
-        to={to}
-      >
-        <Button size="small" variant={variant}>
-          {label}
-        </Button>
-      </Link>
-    )
-}
\ No newline at end of file
+import { makeStyles } from "@material-ui/core/styles";
+import { Button } from "@material-ui/core";
+export const useStyles = makeStyles({
+  title: {
+    marginTop: "8px",
+    marginLeft: "16px",
+  },
+  input: {
```

Comment:
I'm not sure what input is doe. You don't seem to use it.

And why is the other class named title? That's an odd name for a button's class.

And what happens if we put the button underneath another element that hes marginBottom set? now other elements needs to be aware of the margins set up in this file. I'm not sure it makes sense to have the classes here at all.

Reply:
Bump.

Do you use `input` anywhere? I can't see it being used. 
We should try not to add extra bloat to the code. If something isn't used then it shouldn't be there.